docs/src/test.md

```diff
@@ -45,3 +45,72 @@ pytest -svv forge/test/mlir/test_ops.py::test_add
 ```
 
 > - The `-svv` flag is optional and used to display more information about the test run.
+
+## Single operator E2E tests
+
+Single operator E2E tests consists of pre configured collections of in-depth tests for each operator according to test plan.
+Tests include small models consisting of single operator with or without combination with few other operators.
+More details about test plan available on [Test template page](https://github.com/tenstorrent/tt-forge-fe/blob/main/forge/test/operators/test_plan_template.md)
+
+To run all tests from test plan use `test_plan` method:
+
+```sh
+pytest -svv forge/test/operators/pytorch/test_all.py::test_plan
+```
+
+Full list of supported query criteria
+
+| Parameter          | Description                                                                                |
+| ------------------ | ------------------------------------------------------------------------------------------ |
+| OPERATORS          | List of operators                                                                          |
+
+To run single test based on a test id use `test_single` method with `TEST_ID` parameter:
+
+```sh
+TEST_ID='ge-FROM_HOST-None-(1, 2, 3, 4)-Float16_b-HiFi4' pytest -svv forge/test/operators/pytorch/test_all.py::test_single
+```
+
+| Parameter          | Description                                                                                |
+| ------------------ | ------------------------------------------------------------------------------------------ |
+| TEST_ID            | Id of a test containing test parameters                                                    |
+
+To see examples of all available tests use `test_unique` method:
```

Comment:
It'll be good to have a section explaining ID structure. Also, how to find them and create the ones of intereset.

Reply:
Included documentation of test id format in context of `test_unique` and `test_single` commands